File: Problems-and-challenges/project-euler/001/problem001.swift
#!/usr/bin/env xcrun swift

// If we list all the natural numbers below 10 that are multiples of 3 or 5,
// we get 3, 5, 6 and 9. The sum of these multiples is 23.

// Find the sum of all the multiples of 3 or 5 below 1000.

let result = (0...999).filter { $0 % 3 == 0 || $0 % 5 == 0 }.reduce(0, +)
print(result)

File: 001/problem001.js
#!/usr/bin/env node

let list = []
for (let i = 1; i < 1000; i++) {
  if (i % 3 === 0 || i % 5 === 0) {
    list.push(i)
  }
}

const result = list.reduce((sum, current) => sum + current)

console.log(result)
